toolchain/build-toolchain.sh

```diff
@@ -1,69 +1,80 @@
-#!/bin/bash
+#!/usr/bin/env sh
```

Comment:
The entire script is POSIX compliant, and `sh` is symlinked to a POSIX compliant shell on every UNIX-like system that I've ever used. I tend to avoid using `bash` specific code when writing shell scripts to ensure portability. However, I think that `bash` emulates POSIX shell behaviour when invoked as `sh` or with the `--posix` option.

Reply:
I get that, the thing is I'm bound to modify this script at some later point, and I personally won't know what's POSIX compliant or not, so that's a potential source of problem, as opposed to sticking to bash. But that's not really an immediate concern, yeah.

Reply:
Ahh, I see. To that end, this seems to be a good resource. https://mywiki.wooledge.org/Bashism